tests/tests.rs

```diff
@@ -1383,6 +1383,67 @@ fn test_exec() {
     }
 }
 
+#[test]
+fn test_exec_multi() {
+    // TODO test for windows
+    if cfg!(windows) {
+        return;
+    }
```

Comment:
What is the problem here? Only the path separator (`/` vs `\`)? I think we have code somewhere to replace them automatically?

`echo` should be available on Windows as well.

Reply:
I think `echo` is only a shell builtin on Windows

Reply:
These tests seem to indicate otherwise, right?

https://github.com/sharkdp/fd/blob/0fd7ec5c2a5d59616c272fd5438f3187e3685412/tests/tests.rs#L1480-L1501

Reply:
Well, that test fails on my Windows VM.  I bet GIt Bash or MSYS or something installs an `echo.exe` binary that gets picked up on CI.

Reply:
I was just following what we did in the test_exec and test_exec_batch, and (specificaly) test_exec_batch_with_limit.

Reply:
Alright, I'm fine with skipping Windows for now.

In this case, the only open point I see right now is the problem mentioned above (https://github.com/sharkdp/fd/pull/960#issuecomment-1059998122).

Reply:
Oh, you already reported it upstream. Thanks!